src/Result.tsx

```diff
@@ -54,6 +67,45 @@ export default function Result() {
           </tbody>
         </table>
       </div>
+      <div style={{
+        marginTop: '2rem',
+        display: 'flex',
+        justifyContent: 'center',
+        gap: '1rem',
+      }}>
+        <button
+          className='button-primary'
+          onClick={handlePlayAgain}
+          disabled={loading}
+          style={{
+            padding: '0.75rem 1.5rem',
+            fontSize: '1.1rem',
+            borderRadius: '8px',
+            cursor: loading ? 'wait' : 'pointer',
+            opacity: loading ? 0.7 : 1,
+            backgroundColor: '#4CAF50', // Green color for play again
+          }}
+        >
+          {loading ? 'Setting up new game...' : 'Play Again'}
+        </button>
+
+        <button
+          className='button-secondary'
+          onClick={() => ctx.endGame()}
+          disabled={loading}
+          style={{
+            padding: '0.75rem 1.5rem',
+            fontSize: '1.1rem',
+            borderRadius: '8px',
+            cursor: loading ? 'not-allowed' : 'pointer',
+            opacity: loading ? 0.5 : 1,
+            backgroundColor: '#f44336', // Red color for end game
+            color: 'white',
+          }}
+        >
+          End Game
+        </button>
```

Comment:
<img width="456" alt="Screenshot 2025-05-22 at 12 45 50" src="https://github.com/user-attachments/assets/7fc71ad4-2ebc-4e77-9f0f-565256b99388" />

Just use primary and secondary buttons here rather than introducing new colors. Also `End Game` should be on the left and `Play Again` should be on the right.